### PR TITLE
IP list component (Take II)

### PR DESCRIPTION
**Pull request by pfd**
## What do these changes do?

Created Iplist component to render common content under 'IPs to be allowlisted' section

### CODE REVIEW

File: docs/destinations/warehouse-destinations/azure-datalake.mdx [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: "Azure Data Lake"
 description: Step-by-step guide on setting up Azure Data Lake as a destination in RudderStack.
 ---
 
+import Iplist from '/src/components/iplist';
```
Comment (pfd) [inline docs/destinations/warehouse-destinations/azure-datalake.mdx:6]: import the component in `src/@rocketseat/gatsby-theme-docs/components/Docs/index.js` so it's available throughout the docs

File: docs/destinations/warehouse-destinations/azure-synapse.mdx [hunk 2]
```diff
@@ -3,6 +3,8 @@ title: "Azure Synapse"
 description: Step-by-step guide on setting up Azure Synapse as a destination in RudderStack.
 ---
 
+import Iplist from '/src/components/iplist';
```
Comment (pfd) [inline docs/destinations/warehouse-destinations/azure-synapse.mdx:6]: after importing in the theme, you can remove all the import statements in the mdx files

File: src/components/iplist.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+export default function Iplist() {
+  return (
+    <><h2>IPs to be allowlisted</h2>
```
Comment (pfd) [inline src/components/iplist.js:5]: Remove the h2 from the component and add it back to the mdx files. (so it shows up in TOC as we discussed) Probably OK to leave the paragraph text in the component so it's consistent throughout

**Review by pfd — COMMENTED** [review]
A few comments @ishachopra0401 (sorry, i'm the PR author now since i had to redo it) - then n looks great!

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**